Update 00_west_europe.txt

Rhazzazoro opened the pull request:
Changed building set-up for Bohemia. Reshuffled the starting buildings, mainly put manufactories from Branden burg into other states as follows :
University to Anhalt (I thought of the very old Wittenberg university), Glass manufactories to Saxony, Construction sectors to Bohemia proper and Saxony, Gov administration to Silesia, Paper Mills to Moravia.
This is mainly done due to Brandenburg being much less developed in this timeline, so they shouldn't start as one of the most urbanized states, aswell as to combat the starting unemployed issue and make better use of arable land.
Also added one Motor industries to Moravia and 2 Railways to Bohemia proper so that the state starts with sufficient market access

MadJamez commented:
@Rhazzazoro Can you provide more info in the review about what actually changed? And the reasoning behind it.|
Then we can include that in release notes.

MadJamez reviewed (changes requested):
More info about the actual change

Rhazzazoro commented:
Changed building set-up for Bohemia. Reshuffled the starting buildings, mainly put manufactories from Branden burg into other states as follows :
University to Anhalt (I thought of the very old Wittenberg university), Glass manufactories to Saxony, Construction sectors to Bohemia proper and Saxony, Gov administration to Silesia, Paper Mills to Moravia.
This is mainly done due to Brandenburg being much less developed in this timeline, so they shouldn't start as one of the most urbanized states, aswell as to combat the starting unemployed issue and make better use of arable land.
Also added one Motor industries to Moravia and 2 Railways to Bohemia proper so that the state starts with sufficient market access
